### ISSUE TITLE
Create default set of fields

### ISSUE THREAD

**Issue by bootsified** [simplygoodwork/start#9]
There are about 20 fields that I automatically make each time I start a site (most can be imported using FieldManager).  With Neo as an option, I can get by mostly with those default fields.  Something like this:

![screen shot 2018-08-23 at 12 02 49 pm](https://user-images.githubusercontent.com/1693606/44540489-93eb2300-a6cc-11e8-8eac-418928ec826a.png)

**Comment by bootsified** [comment]
If Neo _isn't_ an option...  I don't know if this save as much time.  Never used The Architect before.

**Comment by chrisrowe** [comment]
Yeah, I think FieldManger and The Architect have a lot of overlap, we'll try both before commiting
